assets/package-lock.json

```diff
@@ -8,7 +8,7 @@
       "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
```

Comment:
Is this included in the scope of this PR?

Reply:
That should be included, I will add app key to `.env` too

Reply:
i mean all changes in package-lock. Why we should update much of our dependencies for this PR?

Reply:
I am sorry, salah baca :D